backend/app/services/workflow_engines/reana.py

```diff
@@ -35,13 +35,8 @@ def __init__(self) -> None:
         # self.logger = setup_logging("reana")
         self.logger = logging.getLogger("uvicorn")
 
-    async def ping(self) -> bool:
-        for _ in range(5):
-            out = client.ping(access_token=settings.REANA_ACCESS_TOKEN)
-            if out["error"] is False:
-                return True
-
-        return False
+    async def is_connected(self) -> bool:
```

Comment:
Perhaps `is_available` would be a better name since there is nothing connected actually.